app/assets/stylesheets/modules/countries/_country-widget.scss

```diff
@@ -86,6 +87,22 @@
           background: $white;
           border-bottom: 1px solid transparent;
         }
+        .name {
+          display: block;
+          width: 100%;
+        }
+        .subname {
+          font-size: 10px;
+          display: block;
+          width: 100%;
+          color: #777;
+        }
+        .ipcc {
+          font-size: 9px;
+          display: block;
+          width: 100%;
+          color: #777;
+        }
```

Comment:
Can we change this for a simplest: ...? 

.subname,
.ipcc {
      font-size: 10px;
      display: block;
      width: 100%;
     color: #777;
}
.ipcc {
     font-size: 9px;
 }

And even maybe add .name as well? :lollipop: